packages/dds/map/src/test/mocha/directory.spec.ts

```diff
@@ -1142,7 +1218,7 @@ describe("Directory", () => {
 				);
 			});
 
-			it.skip("Directories should ensure eventual consistency using LWW approach 1", async () => {
+			it("Directories should ensure eventual consistency using LWW approach 1", async () => {
```

Comment:
Update the name/description of the test so it's not the same as the previous one and we can tell the results apart.